features/rni.feature

```diff
@@ -0,0 +1,204 @@
+      | categorie              | Verblijfplaats                                    |
+
+    Scenario: niet leveren RNI-deelnemer uit een categorie waar geen gegevens uit gevraagd zijn
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000036                                   |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                            | waarde                               |
+      | land (13.10)                    | 5010                                 |
+      | RNI-deelnemer (08.88.10)        | 201                                  |
+      | Omschrijving verdrag (08.88.20) | Artikel 45 EU-Werkingsverdrag (VWEU) |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000036                       |
+      | fields              | naam.voornamen                  |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Peter  |
+      En bevat het antwoord GEEN 'rni'
+
+    Scenario: niet leveren RNI-deelnemer voor een categorie waar geen gegeven uit geleverd is
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                            | waarde                                      |
+      | burgerservicenummer             | 000000036                                   |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                  | waarde |
+      | voornamen (02.10)     | Peter  |
+      | geslachtsnaam (02.40) | Jansen |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                            | waarde                               |
+      | land (13.10)                    | 5010                                 |
+      | RNI-deelnemer (08.88.10)        | 201                                  |
+      | Omschrijving verdrag (08.88.20) | Artikel 45 EU-Werkingsverdrag (VWEU) |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                               |
+      | type                | RaadpleegMetBurgerservicenummer                      |
+      | burgerservicenummer | 000000036                                            |
+      | fields              | naam.voornamen,verblijfplaats.verblijfadres.postcode |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Peter  |
+      En bevat het antwoord GEEN 'rni'
```

Comment:
Volgens mij is het beter om de volgende stap te definieren. Dan is de stap ook te gebruiken voor zoek scenario's. Dan kan je meerdere personen terug krijgen met/zonder rni: `En heeft de persoon GEEN 'rni' collectie` of `En heeft de persoon GEEN 'rni' array`. De laatste is conform wat we bij partners, kinderen, ouders en nationaliteit

Reply:
Krijgt je conform met de andere persoon collecties een leeg rni collectie/array als met fields expliciet rni wordt gevraagd?

Reply:
wanneer je niet expliciet om rni vraagt met fields, krijg je geen lege collectie rni. 
wanneer je expliciet om rni vraagt met fields krijg je wel een lege collectie rni. Deze laatste situatie ga ik toevoegen in /dev/rni.feature

Reply:
Het beste is de stap als volgt te herschrijven: `En heeft de persoon GEEN 'rni'`. Als de response meerdere personen heeft is het niet meer duidelijk op welke persoon de stap van toepassing is. Dit is het geval als je personen zoekt. PersoonBeperkt heeft ook de rni property

Reply:
aangepast zoals voorgesteld: En heeft de persoon GEEN 'rni'